toolchain/check/import_cpp.cpp

```diff
@@ -44,6 +44,71 @@ static auto GenerateCppIncludesHeaderCode(
   return code;
 }
 
+namespace {
+
+class CarbonClangDiagnosticConsumer : public clang::DiagnosticConsumer {
+ public:
+  CarbonClangDiagnosticConsumer(Context& context, SemIRLoc loc)
+      : context_(context), loc_(loc) {}
+
+  void HandleDiagnostic(clang::DiagnosticsEngine::Level diag_level,
```

Comment:
We're trying to use trailing returns; clang-tidy won't catch void, but can you please switch this?

Reply:
Sure, done.
My brain still needs to adjust to this style to make sure my eye hurt when it's wrong :)